postman/xs2a gateway.postman_collection.json

```diff
@@ -193,6 +193,81 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "AIS/Consent/G001b",
```

Comment:
That's not a NISP test case name

Reply:
have renamed

Reply:
The test section doesn't have NISP expectations.
Is it possible to also add `AIS/Consents/Create/002`? My consent status test (`AIS/Consents/Status/003a`) depends on it.

Reply:
Let me do it in separate MR

Reply:
added test checks